detect_file.go

```diff
@@ -56,12 +56,6 @@ func fmtFileURL(path string) string {
 	if runtime.GOOS == "windows" {
 		// Make sure we're using "/" on Windows. URLs are "/"-based.
 		path = filepath.ToSlash(path)
-		return fmt.Sprintf("file://%s", path)
 	}
-
-	// Make sure that we don't start with "/" since we add that below.
-	if path[0] == '/' {
-		path = path[1:]
-	}
-	return fmt.Sprintf("file:///%s", path)
```

Comment:
Was this causing a bug ? What makes this change possible ?

Reply:
This was only for telling the client that the FileDetector detected the source string as valid. In the previous code, once this is back to the client, the client was going to remove the forced "file" to use as a key to look for the getter on the map. No longer necessary.

Reply:
Oh right ! 💯  thanks for explaining !